src/java/org/apache/cassandra/tools/nodetool/clustermetadata/InitializeCMS.java

```diff
@@ -26,11 +26,12 @@
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool;
 
-@Command(name = "initializecms", description = "Upgrade from gossip and initialize CMS")
+@Command(name = "initialize", description = "Upgrade from gossip and initialize CMS")
 public class InitializeCMS extends NodeTool.NodeToolCmd
 {
-    @Option(title = "ignored endpoints", name = {"-i", "--ignore"}, description = "Hosts to ignore due to them being down", required = false)
```

Comment:
for a reviewer - this is marked as redundant (in IDEA, at least) because "required" is false by default.